UIT23-Bachelor-Gruppe3/hovedoppgave-gruppe3-UIT-2023
# 4.3 carlos

This branch builds on branch 4.6. **Review 4.6 before this one.**


Changes
- Add RelayConnector to Lobby scene
- Rewrite RelayConnector slightly
- Add create lobby button to Lobby scene
- Create lobby and connect to lobby service on button press

How to test:
Press "Create lobby" button in Lobby Scene and check console log. No errors should occur.
